src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,48 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private Object[] keys;
+    private Object[] values;
+    private int index;
```

Comment:
Rename this variable, so the name will better suit it's purpose.
```suggestion
    private int size;
```